pages/index.tsx

```diff
@@ -152,7 +152,11 @@ const Frontpage: NextPage = () => {
             <Link key={product.id} href={{ pathname: "/assets/[id]", query: { id: product.id } }} title={product.name}>
               <Card elevation="raised">
                 <Card.Header>
-                  <AssetCardTitle as="p">{product.name}</AssetCardTitle>
+                  <AssetCardTitle>
+                    <Typography variant="h4">
```

Comment:
For the front page, I'm adding a h1 for the new title in my PR and changing the "Product" header to h2. So correct level for the card title will then be `h3`

h1 -> h2 -> h3